contracts/BridgeVault.sol

```diff
@@ -2,7 +2,6 @@
 pragma solidity ^0.8.20;
 
 import "@openzeppelin/contracts/access/Ownable.sol";
-import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
we still need this import correct?

Reply:
You are right, we need it, but it is inherited from: `import "./interfaces/IWETH9.sol";`

Reply:
Good point. Let's remove it.